src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -224,6 +258,15 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
     // The _only_ time that's ever appropriate is when the url has changed.
   }, [url]);  // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
+    return autorun(() => {
+      if (phoneRef.current && pluginInfo.textDecorationHandlerInfo) {
+        const textDecorationInfo: ITextDecorationInfo = createTextDecorationInfo();
+        phoneRef.current.post("decorateContent", textDecorationInfo);
+      }
+    });
+  });
```

Comment:
```suggestion
  }, []);
```
I think you still need the empty dependencies array here -- otherwise the effect will run on every render.

Reply:
Fixed.